lib/modules/manager/custom/regex/readme.md

```diff
@@ -1,4 +1,4 @@
-With the `regex` manager you can configure Renovate so it finds dependencies that are not detected by its other built-in package managers.
+With `customManagers` you can configure Renovate so it finds dependencies that are not detected by its other built-in package managers.
```

Comment:
are we shure about this? this doc page is specific to the regex type of custom manager.

Reply:
thoughts on adding a new category to https://docs.renovatebot.com/modules/manager/#supported-managers for example

| Group 	| Category ID|	Managers|
|-|-|-|
| Custom Managers | `custom` | `regex` |